LayoutTests/platform/glib/fast/lists/002-expected.txt

```diff
@@ -6,12 +6,12 @@ layer at (0,0) size 800x600
       RenderBlock {DIV} at (0,0) size 784x36
         RenderBlock {UL} at (0,0) size 784x36
           RenderListItem {LI} at (40,0) size 744x18
-            RenderListMarker at (688,0) size 7x17: bullet
```

Comment:
There is a bunch of test rebaselines in this patch that do not make much sense since they are outside WPT. They should probably be dropped from this patch.